docs/architecture/clients/dependency-injection.md

```diff
@@ -76,6 +79,9 @@ Angular contexts generally use ngModules
 [dependency providers](https://angular.io/guide/dependency-injection-providers) to configure DI. For
 example:
 
+:::danger[DON'T DO THIS] This is a simplified example only - read on to learn about safeProvider.
+:::
+
```

Comment:
❌ issue: prettier does this sometimes because it doesn't understand admonitions, to stop it you have to add some new-lines

```suggestion
:::danger[DON'T DO THIS] 

This is a simplified example only - read on to learn about safeProvider.

:::

```